uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -46,6 +47,31 @@ var Promise = require('es6-promise').Promise;
     });
   });
 
+  /**
+   * Gets a map of contact roles and IDs
+   *
+   * @return {Promise} resolved with {Object}
+   */
+  function getRolesAndIDs () {
+    return new Promise(function (resolve) {
+      var rolesAndIDs = {};
+
+      exec('cv api contact.get', {maxBuffer: 1024 * 1000}, function (err, result) {
```

Comment:
why the `maxBuffer` option?

Reply:
The amount of data that was loading before was not allowed by the default buffer size. Since we reduced it via `return=""` - it works now. However, all depends on the size of the local DB. If it eventually exceeds - the error will be thrown.